<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ludique's Kinect Bundle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>851</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Kinect Utilities</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>511</width>
       <height>207</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>3D single cam viewer(s)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2D viewer(s) and touch settings</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Touch Data extraction : </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>2 Kinect touch Calibration :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="testinfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Touch Extractor and TUIO sender</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Begin 2 kinect position calibration."/>
        </property>
        <property name="text">
         <string>2-Kinect base touch calibration</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>2 Kinect blob viewer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll3d">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>250</width>
           <height>82</height>
          </rect>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>251</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout3d"/>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll2d">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>249</width>
           <height>82</height>
          </rect>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>0</y>
            <width>251</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout2d"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="testUser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>316</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Start skeleton methods"/>
     </property>
     <property name="text">
      <string>Skeleton to TUIO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>24</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Touch methods and camera settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>293</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Skeleton methods</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>821</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>634</x>
       <y>23</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>2 Kinect 3D Viewer</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>20</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="mixViewer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>103</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Mix Viewer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrator3D">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>57</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Calibrator</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>451</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>TUIO Utilities</string>
    </property>
    <widget class="QPushButton" name="tuioMulti">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>51</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplexer, Gestures and Screen Calibration and Mapper</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>201</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imagenes/isotipoLudique.png);
border: 0 none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPruebaGUI">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuPoint_Calibration">
     <property name="title">
      <string>Point Calibration</string>
     </property>
     <addaction name="actionLoadPC"/>
     <addaction name="actionSavePC"/>
    </widget>
    <widget class="QMenu" name="menuCamera_Settings">
     <property name="title">
      <string>Camera Settings</string>
     </property>
     <addaction name="actionLoadCS"/>
     <addaction name="actionSaveCS"/>
    </widget>
    <widget class="QMenu" name="menuGeneral_Settings">
     <property name="title">
      <string>General Settings</string>
     </property>
     <addaction name="actionSaveGS"/>
     <addaction name="actionLoadGS"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="menuPoint_Calibration"/>
    <addaction name="separator"/>
    <addaction name="menuCamera_Settings"/>
    <addaction name="separator"/>
    <addaction name="menuGeneral_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPruebaGUI"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionLoadPC">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSavePC">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionLoadCS">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSaveCS">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionSaveGS">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionLoadGS">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
